### ISSUE TITLE
Registrar: add transfer info

### ISSUE THREAD

**Issue by vohmar** [internetee/registry#1942]
This is for the new portal for registrars

Registrars have requested functionality to request domain contact info prior to actually transferring the domain ie transfer_info in repp using domains transfer code.

**Comment by karlerikounapuu** [comment]
I've got a feeling this endpoint is already present: https://internetee.github.io/repp-apidoc/#get-transfer-info
Returns domain data with all domain's contacts, even for not-sponsoring registrars if they have transfer code

**Comment by vohmar** [comment]
yes it is present in the REPP but not in the portal - our portal users have requested this for the portal as well
